### ISSUE TITLE
Cypress stuck at "opening e2e testing in chrome"

### ISSUE THREAD

**Issue by Jianwei-Wu-1** [cypress-io/cypress#27036]
### Current behavior

Cypress stuck at "opening e2e testing in chrome".

Open Cypress with "./node_modules/.bin/cypress open"

Choose a browser

Cypress stuck at opening with Chrome

<img width="392" alt="Screenshot 2023-06-14 at 10 27 34 AM" src="https://github.com/cypress-io/cypress/assets/27785828/49ebe7ab-3e60-4603-a5f8-891bdfb3207d">

### Desired behavior

Should open with Chrome correctly

### Test code to reproduce

N/A

### Cypress Version

12.14.0

### Node version

v19.8.1

### Operating System

MacOS 13.3.1

**Comment by Jianwei-Wu-1** [comment]
It works OK on Electron. Cypress tests ran correctly.

**Comment by warrensplayer** [comment]
Right now there doesn't seem to be enough information to reproduce the problem on our end. Unless we receive a reliable reproduction, we'll eventually have to close this issue until we can reproduce it. This does not mean that your issue is not happening - it just means that we do not have a path to move forward.

Please provide a reproducible example of the issue you're encountering. Here are some tips for providing a [Short, Self Contained, Correct, Example](http://sscce.org/) and our own [Troubleshooting Cypress](https://on.cypress.io/troubleshooting) guide.

Forking [Cypress Test Tiny](https://github.com/cypress-io/cypress-test-tiny) makes sharing a reproducible example easier to share and easier for our engineers to replicate your issues. This method also keeps the reproduction as simple as possible, which helps us eliminate potential causes and noise from the investigation.

**Comment by Jianwei-Wu-1** [comment]
Thanks for the reply! I think if you use Cypress 12.14.0 with any e2e tests on MacOS 13.3.1 would reproduce this issue. But I'm not sure. Also, the products/tests I'm developing are for internal use, so I'm not sure if I can share on a open-access repo. If I can get approval, I will post a small reproducible example. If not, please feel free to close this or develop your own example to test it out.

**Comment by warrensplayer** [comment]
I am able to run tests with Cypress 12.14.0 and MacOS 13.3.1 just fine.  I will close this for now, but please open another issue if you can create a reproduction.

**Comment by MelaAxel** [comment]
We're seeing this on one machine but in Windows - @warrensplayer do you want a new bug or shall we use this one?

**Comment by Jianwei-Wu-1** [comment]
> We're seeing this on one machine but in Windows - @warrensplayer do you want a new bug or shall we use this one?

Please look into this. This seems to sporadic and hard to figure it out. If not able to be fixed, I plan to move other toolings for e2e testing.

**Comment by warrensplayer** [comment]
@Jianwei-Wu-1 I opened this issue back up so we can continue investigating. I will need some more detail from you in order to figure out what is happening in your scenario.  Could you run Cypress in [debug mode](https://on.cypress.io/troubleshooting#Print-DEBUG-logs) mode and print the entire set of logs here?

Also, are you able to reproduce this scenario with a new simple project, or does this only happen when using Cypress within an existing code repo?  

Any more detail you can provide like the exact steps you used to set up Cypress, a copy of your `cypress.config.js` file, etc. to help us replicate your issue would be most helpful.

**Comment by warrensplayer** [comment]
@MelaAxel Just getting stuck at this step mentioned above could be any number of things, so most likely your issue is not the same.  Please fill out a new bug issue and answer the questions with as much detail as possible.  If you are able to create a simple production, then that would be most helpful of all.  Thanks!

**Comment by OneHatRepo** [comment]
In my case it was a permissions issue on /{user}/Library/Application Support/Cypress in Mac. Giving the proper permissions restored the ability to run tests.

**Comment by Jianwei-Wu-1** [comment]
> @Jianwei-Wu-1 I opened this issue back up so we can continue investigating. I will need some more detail from you in order to figure out what is happening in your scenario. Could you run Cypress in [debug mode](https://on.cypress.io/troubleshooting#Print-DEBUG-logs) mode and print the entire set of logs here?
> 
> Also, are you able to reproduce this scenario with a new simple project, or does this only happen when using Cypress within an existing code repo?
> 
> Any more detail you can provide like the exact steps you used to set up Cypress, a copy of your `cypress.config.js` file, etc. to help us replicate your issue would be most helpful.

Hi @warrensplayer, all of the information I can legally provide is described in the description of this PR. I currently don't have enough time to create a standalone example.

**Comment by MelaAxel** [comment]
Thanks @warrensplayer; created https://github.com/cypress-io/cypress/issues/27091 and included our debug log

**Comment by mike-plummer** [comment]
Similar issues to this have been seen in the past and are typically the result of one of two things:

1. A filesystem permissions issue - without debug logs it's difficult to diagnose if that's what is occurring here
2. A corporate-managed Chrome instance - these often are locked down and restricted to a managed profile which prevents Cypress from opening & automating normally. A common workaround is to download a standalone Chromium install separate from the corporate-managed browser instance, or utilize a headless Chrome instance via Docker

**Comment by Jianwei-Wu-1** [comment]
> Similar issues to this have been seen in the past and are typically the result of one of two things:
> 
> 1. A filesystem permissions issue - without debug logs it's difficult to diagnose if that's what is occurring here
> 2. A corporate-managed Chrome instance - these often are locked down and restricted to a managed profile which prevents Cypress from opening & automating normally. A common workaround is to download a standalone Chromium install separate from the corporate-managed browser instance, or utilize a headless Chrome instance via Docker

Hi Mike, point 1, maybe. point 2, probably not. I can run tests with Chrome a month ago.

**Comment by mike-plummer** [comment]
@Jianwei-Wu-1 Unfortunately we aren't able to know the exact setup of your hardware, installed software, filesystem permissions, etc - we would need the Debug Logs to assist you, or you could enable Debug Logs and interpret any error messages reported therein. Since this scenario is working for the vast majority of Cypress users it is likely this is a permissions or configuration issue unique to your computer. Have you tried asking a coworker if they experience the same behavior? Have you reviewed the Debug Logs to see if there are any hints we could use to diagnose your problem?

**Comment by mike-plummer** [comment]
Unfortunately we have to close this issue due to inactivity. Please comment if there is new information to provide concerning the original issue and we can reopen.

**Comment by talhamit** [comment]
i have the same issue , i,m also stuck on.

![Untitled](https://github.com/cypress-io/cypress/assets/9830460/70859862-a70a-4c23-998b-e83bed1405fe)

**Comment by MikeMcC399** [comment]
@talhamit 

From your screenshot you are using Cypress `12.17.4`, which is a different version from the one in this issue.

You may like to open a new issue and provide debug logs. In almost all cases however this type of error is not due to a bug in Cypress, but due to configuration or local system problems.

There is also a Cypress technical community on Discord available to discuss "how-to" issues:

[![Discord chat](https://img.shields.io/badge/chat-on%20Discord-brightgreen)](https://on.cypress.io/discord)
(click on button)

**Comment by karlamieses** [comment]
Hey @talhamit How did you resolve your issue? I have tried it all:

- [x] Downgrading Cypress
- [x] Checking the permissions as quoted [here](https://github.com/cypress-io/cypress/issues/27036#issuecomment-1599346447)
- [x] Uninstalling and installing Cypress
- [x] Changing the URL that I am using for testing
- [x] Looking into the Cypress extension to make sure is On

I am currently using the most recent version node -v > v21.6.2 and cypress -v >  13.6.6 in my Mac M1 - 14.3.1, and it is not working. I have also checked for the space in my computer and still

**Comment by hal-hive** [comment]
Hi @karlamieses
I had the same issue today and came across this conversation. I found the reason in console of Cypress browser. I have run `cypress open`, then have opened Developer Tools and when I have chosen the chrome browser for testing - there appeared an error in console tab.
```
/home/{user}/.config/Cypress/cy/production/browsers/chrome-stable/interactive/Default/Preferences: Unexpected end of JSON input
```
I use linux Ubuntu 22.04. I have checked this "Preferences" file, it was empty. I suppose it must not be empty.
I have found solution [here](https://docs.cypress.io/guides/references/troubleshooting#Clear-App-Data) in Troubleshooting section of Cypress documentation.
I hope my experience will help you.
